src/main/java/com/google/sps/servlets/GameQuizStatusServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.sps.QuizTimingPropertiesUtils;
+import com.google.sps.UserUtils;
+ 
+//This servlet gets the quiz question from the Game entity 
+//The quiz question is updated if the quiz timestamp is outdated
```

Comment:
Please rewrite this for a reader who doesn't know what a Game entity is. Also, please mention what data is required in the request, and what data will be in the response.

Reply:
I am not sure what to write for the data for the reponse and request because I do not use these parameters in this servlet

Reply:
You use a cookie from the request to determine the current logged-in user. And you write JSON data to the response. It'd be good to document what that JSON data looks like.

Also, the "gets the quiz question from the Game entity" framing only makes sense to you, Hannah, and Emma (at most). Try pretending the reader has used your finished website but hasn't looked at the code before. How would you explain this servlet to that reader?

Reply:
Got it!